x-pack/plugins/cases/public/containers/use_get_cases.tsx

```diff
@@ -131,15 +131,15 @@ export interface UseGetCases extends UseGetCasesState {
 }
 
 export const useGetCases = (
-  initialQueryParams?: QueryParams,
-  initialFilterOptions?: FilterOptions
+  initialQueryParams: Partial<QueryParams> = {},
```

Comment:
changed this so components passing arguments don't need to import `DEFAULT_FILTER_OPTIONS` or `DEFAULT_QUERY_PARAMS`, only need to pass the params that are different from default